README.md

```diff
@@ -36,6 +36,14 @@ region=<set region here>
 integrationcli integrations list -p $project -r $region -t $token
 ```
 
+### Metadata Tokens
+
+If you are using `integrationcli` on Cloud Shell, GCE instances, Cloud Build, then you can use the metadata to get the access token
+
+```sh
+integrationcli integrations list -p $project -r $region --metadata-token
```

Comment:
is the underlying code for `--metadata-token` already available ?

Reply:
nvm, found it. Did not show up in the initial search I did. Then realized I was on the wrong directory lol